webpack.config.js

```diff
@@ -0,0 +1,41 @@
+const _ = require('lodash');
+const path = require('path');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const { tokens } = require('./lib/dsl/lexer');
+
+const allTokenTypeNames = _.mapValues(tokens, tokenType => tokenType.name);
+
+module.exports = {
+  mode: 'production',
+  entry: './lib/dsl/exports.js',
```

Comment:
I am pretty sure webpack handles this automatically, so it would be **redundant**...
See their docs:
https://webpack.js.org/concepts/entry-points/#src/components/Sidebar/Sidebar.jsx

I am using both Mac and Windows for development and would have encountered such 
an issue if it existed.

Reply:
Yes webpack will handle all such stuff

Reply:
Relevant docs: https://webpack.js.org/configuration/watch/#windows-paths